code/modules/shuttle/emergency.dm

```diff
@@ -150,6 +150,7 @@
 	log_game("[key_name(user)] has emagged the emergency shuttle in \
 		[COORD(src)] [time] seconds before launch.")
 	obj_flags |= EMAGGED
+	SSshuttle.emergency.movement_force = list("KNOCKDOWN" = 3, "THROW" = 20, "IGNOREBUCKLE" = TRUE)//YOUR PUNY SEATBELTS CANNOT SAVE YOU NOW, MORTAL
```

Comment:
The ignore buckle part is a bad idea but feel free to keep the increased throw distance.

Reply:
howabout "advanced crash seats" (@Dorsisdwarf) that spawn in the cockpit?

Reply:
![image](https://user-images.githubusercontent.com/29008542/36878484-3543d720-1d8d-11e8-98a0-b2c5b382777d.png)

Reply:
What mso thinks doesn't matter here, removing any ability to avoid the throwback means people have no ability to deal with it when the shuttle is emagged.

It's not too big a deal but I feel it's not a good change, ask kor if they're fine with it.

Reply:
kor didnt seem to sure about it and i dont wanna sprite crash chairs so i give in
:'<